firmware/controllers/actuators/fuel_pump.txt

```diff
@@ -0,0 +1,5 @@
+struct_no_prefix fuel_pump_control_s
+	bit isPrime
+	bit engineTurnedRecently
+	bit isPumpOn
+end_struct
```

Comment:
Does this get us anything, like the ability to view through the console or something?  I didn't see any other files or any indication other parts of the system can find this file.

One downside to bit fields is you can't default them to 0, so Valgrind gets unhappy that you're using uninitialized memory in unit tests, since we instantiate Engine on the stack.  For the real product, it isn't a problem as ChibiOS inits all RAM areas for us.

Reply:
it's supposed to at least eventually